### ISSUE TITLE
ME Level Maintainer (AE2 / LazyAE2)

### ISSUE THREAD

**Issue by Joze97** [Divine-Journey-2/Divine-Journey-2#705]
### Describe the feature

I would like to see an method, to automatically request the crafting of items, like "Refined Storage: Requestify" for Refined Storage. The only mod I know of, that adds this possibility is LazyAE2 with the ME Level Maintainer.

### Reasons why this should be added

1. It would be good for ME Network management, if you can automate the Requests of 5 items per Chanel instead of the current method were you need multiple channels for 1 item.
2. Saves on Space inside of a base.

### Describe alternatives you've considered

There are ways to achieve something similar with OpenComputer. But I'm not aware of any way to do it within the confines of this Mod pack. 

### Additional context

I don't believe the other items that are added by LazyAE2 are needed in the pack.

**Comment by luciscout** [comment]
This already exists in the pack, it’s called a crafting card.

**Comment by KleinGrrmpf** [comment]
Crafting cards are bad tho... 
But I mean, there is this thing called passive automation... Sth. crafted until a drawer is filled up and that drawer is connected to the ME network...

**Comment by WaitingIdly** [comment]
The Level Maintainer functions the same as a crafting card in terms of performance. There is also no need, as this feature can be duplicated through one of many ways, including but not limited to: 
- Using a Level Emitter + Export Bus + Redstone Card + Crafting Card set to only use output from crafts allows you to buffer an arbitrary amount (2 channels per item) - note that this functions the exact same way as as the Refined Storage: Requestify idea.
- Using an Export Bus on round robin + Expansion Card + Crafting Card set to only use output from crafts + limited external storage allows you to buffer an arbitrary amount (1 channel per 9 items).
- Using a ME Interface with a Crafting Card + Storage Bus on the interface allows you to stock up to 9 stacks of a single item, or up to a stack of 9 different types of items (2 channels per 9 items).
- Using a Level Emitter to shut off the crafting of an item after it passes a threshold, either on the machine or the supply of items (1 channel per item).
- Storing in a Drawer (or other inventory, but a Drawer tends to be the best) where you produce it, capping the inventory to the desired amount (a storage bus on the Drawer Controller for like 30+ items).

Note that the first two rely on Crafting Cards + Export Bus, which is rather laggy, and the 3rd option uses an Interface + Crafting Card, which is better but still has issues, and so these options should be used rarely or not at all. The final option is definitely the option that should be gone with.

So, in conclusion: AE2 has the ability to do what you want by default, the Level Maintainer just makes it easier, but doing things this way has performance issues, and so things should be done a different way.
